### ISSUE TITLE
Unable to locate publishable resources

### ISSUE THREAD

**Issue by FelixINX** [rinvex/laravel-statistics#17]
Hello!

When running `php artisan rinvex:publish:statistics`, I get the following message:
```
Unable to locate publishable resources.
Publishing complete.
Unable to locate publishable resources.
Publishing complete.
```

I get the same message when running `php artisan vendor:publish --tag=rinvex-statistics-config` or `php artisan vendor:publish --tag=rinvex-statistics-migrations`

I am using Laravel 7.1.3 and PHP 7.4.3.

**Comment by Omranic** [comment]
Thank you for your report. This has been fixed in the latest v4.0.1 release.

**Comment by FelixINX** [comment]
Thanks @Omranic for the quick fix :+1:
